static/js/components/FacultyCarousel.js

```diff
@@ -19,11 +19,15 @@ export default class FacultyCarousel extends React.Component {
       responsive:     [
         {
           breakpoint: 600,
-          settings:   "unslick"
+          settings:   {
+            slidesToShow: 1
+          }
         },
         {
-          breakpoint: 800,
-          settings:   "unslick"
+          breakpoint: 840,
```

Comment:
Are we standardizing on a tablet breakpoint of 840px?

Reply:
I updated it based on the media queries we usually use in the styling e.g. [this](https://github.com/mitodl/micromasters/blob/master/static/scss/program-page.scss#L417). And this slick setting will be applied on screens from `840px` and below until it reaches the lower threshold of `600px` In our case.